docs/en/operations/system-tables.md

```diff
@@ -143,6 +143,88 @@ This table contains a single String column called ‘name’ – the name of a d
 Each database that the server knows about has a corresponding entry in the table.
 This system table is used for implementing the `SHOW DATABASES` query.
 
+## system.data\_type-families {#system_tables-data_type_families}
+
+Contains information about properties of SQL data types.
+
+Columns:
+
+-   `name` ([String](../sql-reference/data-types/string.md)) — Data type name.
+-   `case_insensitive` ([UInt8](../sql-reference/data-types/int-uint.md)) — Shows whether a data type is case sensetive.
```

Comment:
```suggestion
-   `case_insensitive` ([UInt8](../sql-reference/data-types/int-uint.md)) — Property that shows whether you can use a data type name in a query in case insensitive manner or not. For example, `Date` and `date` are valid both.
```